tests/integration/operator/operator_integration_test.go

```diff
@@ -0,0 +1,97 @@
+package operator
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/k8ssandra/k8ssandra/tests/integration/util"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	suiteName           = "k8ssandra-integration-suite"
+	operatorReleaseName = "k8ssandra"
+	namespace           = "k8ssandra"
+	clusterReleaseName  = "cassdc"
+	operatorChart       = "../../../charts/k8ssandra"
+	clusterChart        = "../../../charts/k8ssandra-cluster"
+)
+
+func Test(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, suiteName)
+}
+
+// setup used for creating test option context and cleanup of operator and cluster.
+func setup() K8ssandraOptionsContext {
+
+	kubeOptions := k8s.NewKubectlOptions("", "", namespace)
+	operatorOptions := &helm.Options{KubectlOptions: kubeOptions}
+	clusterOptions := &helm.Options{KubectlOptions: kubeOptions}
+
+	ctx := CreateK8ssandraOptionsContext(operatorOptions, clusterOptions)
+	UninstallRelease(ctx.Cluster, clusterReleaseName)
+	UninstallRelease(ctx.Operator, operatorReleaseName)
+	return ctx
+}
+
+var _ = Describe(suiteName, func() {
+
+	var options K8ssandraOptionsContext
+
+	Describe("baseline required preconditions", func() {
```

Comment:
Is this scenario redundant since we already have to install the k8ssandra chart in the `k8ssandra cluster integration` scenario below?

Reply:
Not the entire portion as it verifies that the test setup conditions are valid.  If something changed to the setup function logic, this would catch that.  As we add more test function (edge cases, etc.) I suspect we don't want to perform this to ensure our options are setup correctly.

```go
By("Expecting operator and cluster options to be setup")
Ω(options).ShouldNot(BeNil())
Ω(options.Cluster).ShouldNot(BeNil())
Ω(options.Operator).ShouldNot(BeNil())
```

I agree, we can remove the following portion as this is now covered in the integration with the cass-cluster.

```go
InstallChart(options.Operator, operatorChart, operatorReleaseName, "")
By("Expecting the operator to be in a deployed status")
Ω(IsReleaseDeployed(options.Operator, "k8ssandra")).Should(BeTrue())
```

Reply:
The  InstallChart for operator has been removed from this setup check as part of the latest update.

The setup will continue to verify that the options are created as expected to support the tests scoped to this package.